Sources/Zephyr.swift

```diff
@@ -551,6 +551,6 @@ private extension Zephyr {
 
     // Notify any observers that we have finished synchronizing an update from iCloud.
     static func postNotificationAfterSyncFromCloud() {
-        NotificationCenter.default.post(name: Zephyr.keysDidChangeOnCloudNotification, object: nil)
+        NotificationCenter.default.post(name: keysDidChangeOnCloudNotification, object: nil)
```

Comment:
What am I missing here? Why is removing the Struct prefix something I should do?

Reply:
Hi Arthur,

It’s a suggestion that made my project function correct when implementing your library. Since public static let keysDidChangeOnCloudNotification = Notification.Name("ZephyrKeysDidChangeOnCloudNotification") Is defined at line 35 there would be no need for a reference like .self? Because Notification.Name("<<name>>") is the correct syntax.

I imported the zephyr file direct into my project, when using the cocoabuild the reference ‘Zephyr.’ might be useful. The reference to Zephyr is needed in the appdelegate to refer to the correct variable ofcourse.

Thanks again for making this, it’s easy to implement and great improved functionality over standard userdefaults use.

Or am I missing something?

Reply:
If it is somehow possible to additionally identify only the updated/changed keys after synchronisation then it would for instance be possible to add a method, via an observer (keysDidChangeOnCloudNotification) to do actions specifically for only the changed data (like reschedule notifications)